package.json

```diff
@@ -2,13 +2,13 @@
   "scripts": {
     "reset": "lerna clean --yes && npm run bootstrap && npm run build-packages",
     "bootstrap": "lerna bootstrap && npm run bootstrap-nextjs-sample",
+    "bootstrap:ci": "lerna bootstrap --ci",
```

Comment:
@ambrauer Do we need to bootstrap packages+samples? Previously we only bootstrapped packages using: `bootstrap-packages:ci`

Reply:
@sc-illiakovalenko Yeah, the samples need to be bootstrapped as well in order to lint them.